src/pyfme/aircrafts/aircraft.py

```diff
@@ -8,43 +8,39 @@
 
 """
 from abc import abstractmethod
-from copy import deepcopy
-from warnings import warn
 
 import numpy as np
-from scipy.optimize import least_squares
 
-from pyfme.environment.environment import Environment
```

Comment:
These imports were not used before? There should be some quality check in place then I think.

Reply:
I have had a look at many style and code quality aspects in latest revisions. Thanks for reporting!